[2023-12-11T02:13:55.770+0000] {processor.py:154} INFO - Started process (PID=1159) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:13:55.771+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:13:55.773+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:13:55.772+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:13:55.802+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:13:55.799+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 5
    )
    ^
SyntaxError: invalid syntax
[2023-12-11T02:13:55.803+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:13:55.836+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.069 seconds
[2023-12-11T02:14:22.462+0000] {processor.py:154} INFO - Started process (PID=1209) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:14:22.464+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:14:22.466+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:14:22.465+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:14:22.513+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:14:22.512+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 7
    )
    ^
SyntaxError: invalid syntax
[2023-12-11T02:14:22.514+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:14:22.548+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.095 seconds
[2023-12-11T02:14:46.800+0000] {processor.py:154} INFO - Started process (PID=1259) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:14:46.801+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:14:46.802+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:14:46.802+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:14:46.839+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:14:46.837+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 7
    )
    ^
SyntaxError: invalid syntax
[2023-12-11T02:14:46.841+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:14:46.877+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.081 seconds
[2023-12-11T02:15:17.415+0000] {processor.py:154} INFO - Started process (PID=1314) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:15:17.418+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:15:17.424+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:15:17.424+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:15:17.464+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:15:17.462+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 7
    )
    ^
SyntaxError: invalid syntax
[2023-12-11T02:15:17.465+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:15:17.514+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.103 seconds
[2023-12-11T02:15:47.913+0000] {processor.py:154} INFO - Started process (PID=1378) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:15:47.914+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:15:47.915+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:15:47.915+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:15:47.949+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:15:47.947+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 7
    )
    ^
SyntaxError: invalid syntax
[2023-12-11T02:15:47.950+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:15:47.979+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.074 seconds
[2023-12-11T02:16:12.376+0000] {processor.py:154} INFO - Started process (PID=1431) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:16:12.378+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:16:12.380+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:16:12.379+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:16:12.419+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:16:12.416+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 8
    )
    ^
SyntaxError: invalid syntax
[2023-12-11T02:16:12.420+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:16:12.458+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.088 seconds
[2023-12-11T02:16:29.175+0000] {processor.py:154} INFO - Started process (PID=1456) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:16:29.178+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:16:29.182+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:16:29.181+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:16:29.219+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:16:29.217+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 9
    )
    ^
SyntaxError: invalid syntax
[2023-12-11T02:16:29.220+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:16:29.250+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.078 seconds
[2023-12-11T02:16:35.278+0000] {processor.py:154} INFO - Started process (PID=1464) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:16:35.279+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:16:35.280+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:16:35.280+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:16:35.320+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:16:35.318+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 9
    )
    ^
SyntaxError: invalid syntax
[2023-12-11T02:16:35.321+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:16:35.369+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.099 seconds
[2023-12-11T02:17:05.665+0000] {processor.py:154} INFO - Started process (PID=1528) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:17:05.668+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:17:05.671+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:17:05.670+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:17:05.708+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:17:05.707+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 9
    )
    ^
SyntaxError: invalid syntax
[2023-12-11T02:17:05.709+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:17:05.740+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.079 seconds
[2023-12-11T02:17:35.984+0000] {processor.py:154} INFO - Started process (PID=1582) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:17:35.985+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:17:35.987+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:17:35.986+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:17:36.014+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:17:36.010+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 9
    )
    ^
SyntaxError: invalid syntax
[2023-12-11T02:17:36.017+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:17:36.060+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.080 seconds
[2023-12-11T02:18:06.692+0000] {processor.py:154} INFO - Started process (PID=1646) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:18:06.694+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:18:06.697+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:18:06.697+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:18:06.740+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:18:06.739+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 9
    )
    ^
SyntaxError: invalid syntax
[2023-12-11T02:18:06.741+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:18:06.776+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.097 seconds
[2023-12-11T02:18:26.672+0000] {processor.py:154} INFO - Started process (PID=1678) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:18:26.675+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:18:26.678+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:18:26.678+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:18:26.782+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:18:27.036+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:18:27.036+0000] {manager.py:507} INFO - Created Permission View: can delete on DAG:user_processing
[2023-12-11T02:18:27.062+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:18:27.062+0000] {manager.py:507} INFO - Created Permission View: can edit on DAG:user_processing
[2023-12-11T02:18:27.079+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:18:27.079+0000] {manager.py:507} INFO - Created Permission View: can read on DAG:user_processing
[2023-12-11T02:18:27.081+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:18:27.080+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:18:27.106+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:18:27.105+0000] {dag.py:2606} INFO - Creating ORM DAG for user_processing
[2023-12-11T02:18:27.137+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:18:27.136+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:18:27.187+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.519 seconds
[2023-12-11T02:18:57.686+0000] {processor.py:154} INFO - Started process (PID=1742) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:18:57.688+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:18:57.691+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:18:57.690+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:18:57.791+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:18:57.855+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:18:57.854+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:18:57.902+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:18:57.901+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:18:57.944+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.270 seconds
[2023-12-11T02:19:28.115+0000] {processor.py:154} INFO - Started process (PID=1797) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:19:28.116+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:19:28.118+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:19:28.117+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:19:28.164+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:19:28.222+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:19:28.222+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:19:28.261+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:19:28.260+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:19:28.291+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.180 seconds
[2023-12-11T02:19:58.872+0000] {processor.py:154} INFO - Started process (PID=1861) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:19:58.874+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:19:58.875+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:19:58.875+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:19:58.918+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:19:58.979+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:19:58.978+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:19:59.029+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:19:59.029+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:19:59.064+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.197 seconds
[2023-12-11T02:20:25.340+0000] {processor.py:154} INFO - Started process (PID=1911) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:20:25.341+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:20:25.342+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:20:25.342+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:20:25.513+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:20:25.583+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:20:25.582+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:20:25.638+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:20:25.638+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:20:25.715+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.379 seconds
[2023-12-11T02:20:55.916+0000] {processor.py:154} INFO - Started process (PID=1966) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:20:55.917+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:20:55.919+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:20:55.919+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:20:56.023+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:20:56.067+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:20:56.067+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:20:56.110+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:20:56.110+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:20:56.137+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.226 seconds
[2023-12-11T02:21:26.282+0000] {processor.py:154} INFO - Started process (PID=2030) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:21:26.284+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:21:26.285+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:21:26.285+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:21:26.430+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:21:26.492+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:21:26.492+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:21:26.540+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:21:26.539+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:21:26.571+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.295 seconds
[2023-12-11T02:21:56.763+0000] {processor.py:154} INFO - Started process (PID=2085) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:21:56.765+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:21:56.766+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:21:56.766+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:21:56.900+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:21:56.950+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:21:56.950+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:21:56.994+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:21:56.994+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:21:57.026+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.267 seconds
[2023-12-11T02:22:27.088+0000] {processor.py:154} INFO - Started process (PID=2149) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:22:27.090+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:22:27.091+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:22:27.091+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:22:27.213+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:22:27.270+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:22:27.270+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:22:27.321+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:22:27.320+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:22:27.358+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.275 seconds
[2023-12-11T02:22:57.555+0000] {processor.py:154} INFO - Started process (PID=2204) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:22:57.557+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:22:57.559+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:22:57.559+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:22:57.684+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:22:57.745+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:22:57.745+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:22:57.792+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:22:57.792+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:22:57.839+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.290 seconds
[2023-12-11T02:23:28.034+0000] {processor.py:154} INFO - Started process (PID=2269) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:23:28.035+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:23:28.037+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:23:28.037+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:23:28.171+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:23:28.217+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:23:28.217+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:23:28.263+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:23:28.263+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:23:28.291+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.262 seconds
[2023-12-11T02:23:58.563+0000] {processor.py:154} INFO - Started process (PID=2324) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:23:58.564+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:23:58.566+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:23:58.566+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:23:58.691+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:23:58.737+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:23:58.737+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:23:58.791+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:23:58.791+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:23:58.822+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.264 seconds
[2023-12-11T02:24:28.885+0000] {processor.py:154} INFO - Started process (PID=2388) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:24:28.887+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:24:28.888+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:24:28.888+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:24:29.032+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:24:29.088+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:24:29.087+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:24:29.130+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:24:29.130+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:24:29.163+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.283 seconds
[2023-12-11T02:24:31.180+0000] {processor.py:154} INFO - Started process (PID=2390) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:24:31.182+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:24:31.183+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:24:31.183+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:24:31.337+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:24:31.475+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:24:31.474+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:24:31.527+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:24:31.526+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:24:31.609+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.434 seconds
[2023-12-11T02:25:02.270+0000] {processor.py:154} INFO - Started process (PID=2454) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:25:02.271+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:25:02.273+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:25:02.273+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:25:02.412+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:25:02.470+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:25:02.470+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:25:02.539+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:25:02.539+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:25:02.582+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.316 seconds
[2023-12-11T02:25:32.682+0000] {processor.py:154} INFO - Started process (PID=2509) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:25:32.686+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:25:32.688+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:25:32.687+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:25:32.813+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:25:32.878+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:25:32.878+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:25:32.925+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:25:32.925+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:25:32.959+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.283 seconds
[2023-12-11T02:26:03.218+0000] {processor.py:154} INFO - Started process (PID=2573) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:26:03.220+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:26:03.222+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:26:03.221+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:26:03.346+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:26:03.397+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:26:03.397+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:26:03.435+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:26:03.435+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:26:03.464+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.250 seconds
[2023-12-11T02:26:34.370+0000] {processor.py:154} INFO - Started process (PID=2628) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:26:34.372+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:26:34.374+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:26:34.374+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:26:34.496+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:26:34.557+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:26:34.557+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:26:34.600+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:26:34.600+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:26:34.634+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.269 seconds
[2023-12-11T02:27:05.431+0000] {processor.py:154} INFO - Started process (PID=2692) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:27:05.432+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:27:05.435+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:27:05.435+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:27:05.563+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:27:05.627+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:27:05.627+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:27:05.669+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:27:05.668+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:27:05.704+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.277 seconds
[2023-12-11T02:27:35.865+0000] {processor.py:154} INFO - Started process (PID=2747) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:27:35.866+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:27:35.868+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:27:35.868+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:27:35.988+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:27:36.043+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:27:36.042+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:27:36.091+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:27:36.090+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:27:36.125+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.265 seconds
[2023-12-11T02:28:06.756+0000] {processor.py:154} INFO - Started process (PID=2811) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:28:06.757+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:28:06.759+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:28:06.759+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:28:06.897+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:28:06.972+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:28:06.972+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:28:07.061+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:28:07.060+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:28:07.105+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.355 seconds
[2023-12-11T02:28:37.273+0000] {processor.py:154} INFO - Started process (PID=2866) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:28:37.276+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:28:37.278+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:28:37.277+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:28:37.403+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:28:37.483+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:28:37.483+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:28:37.521+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:28:37.520+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:28:37.551+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.283 seconds
[2023-12-11T02:29:08.066+0000] {processor.py:154} INFO - Started process (PID=2930) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:29:08.069+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:29:08.070+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:29:08.070+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:29:08.205+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:29:08.256+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:29:08.256+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:29:08.295+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:29:08.295+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:29:08.328+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.268 seconds
[2023-12-11T02:29:38.756+0000] {processor.py:154} INFO - Started process (PID=2985) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:29:38.757+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:29:38.758+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:29:38.758+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:29:38.872+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:29:38.934+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:29:38.934+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:29:38.979+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:29:38.979+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:29:39.015+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.264 seconds
[2023-12-11T02:30:09.212+0000] {processor.py:154} INFO - Started process (PID=3050) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:30:09.214+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:30:09.215+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:30:09.215+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:30:09.331+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:30:09.383+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:30:09.383+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:30:09.425+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:30:09.425+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:30:09.461+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.256 seconds
[2023-12-11T02:30:40.059+0000] {processor.py:154} INFO - Started process (PID=3105) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:30:40.060+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:30:40.062+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:30:40.062+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:30:40.199+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:30:40.253+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:30:40.253+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:30:40.295+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:30:40.295+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:30:40.330+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.277 seconds
[2023-12-11T02:31:00.471+0000] {processor.py:154} INFO - Started process (PID=3156) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:31:00.472+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:31:00.474+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:31:00.473+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:31:00.605+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:31:00.599+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:31:00.608+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:31:00.634+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.167 seconds
[2023-12-11T02:31:30.905+0000] {processor.py:154} INFO - Started process (PID=3220) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:31:30.907+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:31:30.909+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:31:30.908+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:31:31.029+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:31:31.022+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:31:31.031+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:31:31.061+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.161 seconds
[2023-12-11T02:32:01.496+0000] {processor.py:154} INFO - Started process (PID=3276) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:01.498+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:32:01.499+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:32:01.499+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:01.617+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:32:01.609+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:32:01.619+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:01.649+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.156 seconds
[2023-12-11T02:32:02.709+0000] {processor.py:154} INFO - Started process (PID=3278) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:02.711+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:32:02.713+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:32:02.713+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:02.883+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:32:02.877+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:32:02.886+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:02.918+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.214 seconds
[2023-12-11T02:32:05.749+0000] {processor.py:154} INFO - Started process (PID=3289) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:05.750+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:32:05.752+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:32:05.752+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:05.896+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:32:05.889+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:32:05.899+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:05.931+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.187 seconds
[2023-12-11T02:32:21.048+0000] {processor.py:154} INFO - Started process (PID=3319) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:21.050+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:32:21.051+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:32:21.051+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:21.187+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:32:21.178+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:32:21.189+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:21.215+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.171 seconds
[2023-12-11T02:32:45.498+0000] {processor.py:154} INFO - Started process (PID=3365) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:45.499+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:32:45.500+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:32:45.500+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:45.639+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:32:45.632+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:32:45.641+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:32:45.666+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.171 seconds
[2023-12-11T02:33:15.775+0000] {processor.py:154} INFO - Started process (PID=3420) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:33:15.776+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:33:15.777+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:33:15.777+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:33:15.896+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:33:15.888+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:33:15.898+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:33:15.923+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.152 seconds
[2023-12-11T02:33:39.692+0000] {processor.py:154} INFO - Started process (PID=3465) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:33:39.694+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:33:39.695+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:33:39.695+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:33:39.903+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:33:39.888+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:33:39.906+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:33:39.951+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.264 seconds
[2023-12-11T02:34:10.607+0000] {processor.py:154} INFO - Started process (PID=3529) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:34:10.608+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:34:10.610+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:34:10.609+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:34:10.723+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:34:10.716+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:34:10.726+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:34:10.757+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.154 seconds
[2023-12-11T02:34:41.258+0000] {processor.py:154} INFO - Started process (PID=3584) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:34:41.259+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:34:41.261+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:34:41.260+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:34:41.400+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:34:41.393+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:34:41.403+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:34:41.437+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.183 seconds
[2023-12-11T02:34:47.008+0000] {processor.py:154} INFO - Started process (PID=3600) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:34:47.010+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:34:47.012+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:34:47.012+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:34:47.171+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:34:47.165+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:34:47.175+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:34:47.202+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.201 seconds
[2023-12-11T02:35:17.590+0000] {processor.py:154} INFO - Started process (PID=3655) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:35:17.592+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:35:17.594+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:35:17.594+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:35:17.722+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:35:17.712+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:35:17.732+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:35:17.763+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.179 seconds
[2023-12-11T02:35:46.584+0000] {processor.py:154} INFO - Started process (PID=3714) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:35:46.585+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:35:46.587+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:35:46.587+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:35:46.737+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:35:46.726+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:35:46.745+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:35:46.798+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.219 seconds
[2023-12-11T02:36:16.972+0000] {processor.py:154} INFO - Started process (PID=3769) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:36:16.973+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:36:16.976+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:36:16.975+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:36:17.145+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:36:17.137+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:36:17.148+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:36:17.177+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.209 seconds
[2023-12-11T02:36:47.245+0000] {processor.py:154} INFO - Started process (PID=3833) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:36:47.248+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:36:47.249+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:36:47.249+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:36:47.403+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:36:47.393+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:36:47.405+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:36:47.429+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.190 seconds
[2023-12-11T02:37:17.941+0000] {processor.py:154} INFO - Started process (PID=3888) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:37:17.943+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:37:17.945+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:37:17.945+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:37:18.080+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:37:18.066+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:37:18.085+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:37:18.146+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.210 seconds
[2023-12-11T02:37:20.294+0000] {processor.py:154} INFO - Started process (PID=3900) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:37:20.295+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:37:20.297+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:37:20.296+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:37:20.331+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:37:20.330+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 50
    )
    ^
SyntaxError: invalid syntax
[2023-12-11T02:37:20.332+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:37:20.357+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.069 seconds
[2023-12-11T02:37:31.945+0000] {processor.py:154} INFO - Started process (PID=3927) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:37:31.946+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:37:31.947+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:37:31.947+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:37:32.081+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:37:32.075+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:37:32.083+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:37:32.110+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.170 seconds
[2023-12-11T02:38:02.409+0000] {processor.py:154} INFO - Started process (PID=3990) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:38:02.411+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:38:02.412+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:38:02.412+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:38:02.533+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:38:02.525+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:38:02.536+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:38:02.570+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.166 seconds
[2023-12-11T02:38:32.685+0000] {processor.py:154} INFO - Started process (PID=4045) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:38:32.687+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:38:32.688+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:38:32.688+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:38:32.802+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:38:32.794+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:38:32.804+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:38:32.836+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.155 seconds
[2023-12-11T02:39:03.149+0000] {processor.py:154} INFO - Started process (PID=4109) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:39:03.151+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:39:03.152+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:39:03.152+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:39:03.272+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:39:03.264+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:39:03.275+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:39:03.304+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.160 seconds
[2023-12-11T02:39:34.079+0000] {processor.py:154} INFO - Started process (PID=4164) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:39:34.081+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:39:34.083+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:39:34.083+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:39:34.211+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:39:34.204+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:39:34.213+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:39:34.244+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.172 seconds
[2023-12-11T02:40:04.491+0000] {processor.py:154} INFO - Started process (PID=4228) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:40:04.493+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:40:04.495+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:40:04.495+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:40:04.651+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:40:04.638+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:40:04.657+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:40:04.695+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.209 seconds
[2023-12-11T02:40:34.979+0000] {processor.py:154} INFO - Started process (PID=4290) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:40:34.981+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:40:34.982+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:40:34.982+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:40:35.103+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:40:35.096+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:40:35.105+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:40:35.130+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.156 seconds
[2023-12-11T02:41:01.436+0000] {processor.py:154} INFO - Started process (PID=4339) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:41:01.437+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:41:01.439+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:41:01.439+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:41:01.565+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:41:01.558+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:41:01.568+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:41:01.596+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.163 seconds
[2023-12-11T02:41:31.737+0000] {processor.py:154} INFO - Started process (PID=4395) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:41:31.739+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:41:31.740+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:41:31.740+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:41:31.871+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:41:31.863+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:41:31.873+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:41:31.901+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.168 seconds
[2023-12-11T02:42:02.031+0000] {processor.py:154} INFO - Started process (PID=4458) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:42:02.033+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:42:02.034+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:42:02.034+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:42:02.147+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:42:02.140+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:42:02.150+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:42:02.180+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.154 seconds
[2023-12-11T02:42:32.551+0000] {processor.py:154} INFO - Started process (PID=4513) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:42:32.552+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:42:32.554+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:42:32.553+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:42:32.680+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:42:32.671+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:42:32.682+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:42:32.707+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.161 seconds
[2023-12-11T02:42:33.885+0000] {processor.py:154} INFO - Started process (PID=4517) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:42:33.887+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:42:33.889+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:42:33.888+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:42:34.020+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:42:34.013+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:42:34.022+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:42:34.051+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.170 seconds
[2023-12-11T02:43:04.312+0000] {processor.py:154} INFO - Started process (PID=4580) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:43:04.314+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:43:04.316+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:43:04.315+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:43:04.433+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:43:04.424+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:43:04.436+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:43:04.472+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.165 seconds
[2023-12-11T02:43:14.969+0000] {processor.py:154} INFO - Started process (PID=4603) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:43:14.970+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:43:14.972+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:43:14.971+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:43:15.108+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:43:15.101+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:43:15.111+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:43:15.149+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.184 seconds
[2023-12-11T02:43:45.469+0000] {processor.py:154} INFO - Started process (PID=4658) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:43:45.471+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:43:45.473+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:43:45.473+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:43:45.609+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:43:45.600+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:43:45.612+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:43:45.641+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.177 seconds
[2023-12-11T02:44:15.815+0000] {processor.py:154} INFO - Started process (PID=4722) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:44:15.817+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:44:15.818+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:44:15.818+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:44:15.949+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:44:15.940+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:44:15.952+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:44:15.985+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.173 seconds
[2023-12-11T02:44:46.599+0000] {processor.py:154} INFO - Started process (PID=4777) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:44:46.601+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:44:46.603+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:44:46.602+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:44:46.722+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:44:46.716+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:44:46.724+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:44:46.754+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.159 seconds
[2023-12-11T02:44:50.834+0000] {processor.py:154} INFO - Started process (PID=4788) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:44:50.836+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:44:50.837+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:44:50.837+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:44:50.978+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:44:50.971+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:44:50.981+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:44:51.020+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.193 seconds
[2023-12-11T02:45:21.692+0000] {processor.py:154} INFO - Started process (PID=4843) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:45:21.694+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:45:21.697+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:45:21.696+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:45:21.828+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:45:21.821+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:45:21.831+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:45:21.857+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.170 seconds
[2023-12-11T02:45:52.072+0000] {processor.py:154} INFO - Started process (PID=4907) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:45:52.073+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:45:52.074+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:45:52.074+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:45:52.187+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:45:52.179+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:45:52.190+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:45:52.220+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.152 seconds
[2023-12-11T02:46:22.409+0000] {processor.py:154} INFO - Started process (PID=4962) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:46:22.411+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:46:22.412+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:46:22.412+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:46:22.524+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:46:22.516+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:46:22.527+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:46:22.557+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.152 seconds
[2023-12-11T02:46:45.996+0000] {processor.py:154} INFO - Started process (PID=5016) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:46:45.998+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:46:46.000+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:46:45.999+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:46:46.139+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:46:46.131+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:46:46.141+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:46:46.174+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.182 seconds
[2023-12-11T02:47:16.369+0000] {processor.py:154} INFO - Started process (PID=5079) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:47:16.371+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:47:16.372+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:47:16.372+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:47:16.502+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:47:16.494+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:47:16.504+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:47:16.531+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.168 seconds
[2023-12-11T02:47:47.098+0000] {processor.py:154} INFO - Started process (PID=5134) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:47:47.104+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:47:47.106+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:47:47.106+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:47:47.277+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:47:47.270+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:47:47.280+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:47:47.305+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.212 seconds
[2023-12-11T02:48:17.526+0000] {processor.py:154} INFO - Started process (PID=5198) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:48:17.527+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:48:17.529+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:48:17.529+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:48:17.654+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:48:17.646+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:48:17.656+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:48:17.683+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.162 seconds
[2023-12-11T02:48:39.298+0000] {processor.py:154} INFO - Started process (PID=5233) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:48:39.300+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:48:39.302+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:48:39.302+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:48:39.451+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:48:39.444+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:48:39.454+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:48:39.481+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.186 seconds
[2023-12-11T02:48:39.790+0000] {processor.py:154} INFO - Started process (PID=5236) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:48:39.791+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:48:39.793+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:48:39.792+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:48:39.907+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:48:39.901+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:48:39.910+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:48:39.934+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.148 seconds
[2023-12-11T02:49:08.702+0000] {processor.py:154} INFO - Started process (PID=5298) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:49:08.703+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:49:08.705+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:49:08.705+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:49:08.857+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:49:08.846+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:49:08.859+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:49:08.891+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.195 seconds
[2023-12-11T02:49:39.452+0000] {processor.py:154} INFO - Started process (PID=5353) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:49:39.453+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:49:39.455+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:49:39.454+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:49:39.590+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:49:39.583+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:49:39.592+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:49:39.621+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.173 seconds
[2023-12-11T02:50:04.242+0000] {processor.py:154} INFO - Started process (PID=5390) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:50:04.244+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:50:04.246+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:50:04.245+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:50:04.408+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:50:04.399+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:50:04.411+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:50:04.439+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.204 seconds
[2023-12-11T02:50:34.877+0000] {processor.py:154} INFO - Started process (PID=5454) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:50:34.879+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:50:34.881+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:50:34.881+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:50:34.993+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:50:34.985+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:50:34.996+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:50:35.027+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.153 seconds
[2023-12-11T02:50:53.394+0000] {processor.py:154} INFO - Started process (PID=5496) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:50:53.395+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:50:53.396+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:50:53.396+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:50:53.535+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:50:53.527+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:50:53.538+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:50:53.571+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.182 seconds
[2023-12-11T02:50:56.638+0000] {processor.py:154} INFO - Started process (PID=5507) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:50:56.640+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:50:56.641+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:50:56.641+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:50:56.786+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:50:56.779+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:50:56.787+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:50:56.813+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.180 seconds
[2023-12-11T02:51:15.887+0000] {processor.py:154} INFO - Started process (PID=5546) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:51:15.889+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:51:15.891+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:51:15.891+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:51:16.046+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:51:16.037+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:51:16.049+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:51:16.079+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.198 seconds
[2023-12-11T02:51:46.284+0000] {processor.py:154} INFO - Started process (PID=5610) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:51:46.286+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:51:46.288+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:51:46.287+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:51:46.401+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:51:46.393+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:51:46.403+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:51:46.429+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.150 seconds
[2023-12-11T02:52:16.642+0000] {processor.py:154} INFO - Started process (PID=5665) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:52:16.643+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:52:16.645+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:52:16.644+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:52:16.769+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:52:16.761+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:52:16.772+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:52:16.806+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.173 seconds
[2023-12-11T02:52:47.345+0000] {processor.py:154} INFO - Started process (PID=5729) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:52:47.346+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:52:47.348+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:52:47.347+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:52:47.462+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:52:47.454+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3, in <module>
    from airflow.providers.http.sensor.http import HttpSensor
ModuleNotFoundError: No module named 'airflow.providers.http.sensor'
[2023-12-11T02:52:47.465+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:52:47.490+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.149 seconds
[2023-12-11T02:52:55.393+0000] {processor.py:154} INFO - Started process (PID=5736) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:52:55.394+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:52:55.396+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:52:55.395+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:52:57.401+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:52:57.769+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:52:57.769+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:52:57.808+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:52:57.808+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:52:57.851+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 2.464 seconds
[2023-12-11T02:53:27.987+0000] {processor.py:154} INFO - Started process (PID=5803) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:53:27.989+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:53:27.990+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:53:27.990+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:53:28.676+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:53:28.978+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:53:28.977+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:53:29.022+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:53:29.022+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:53:29.053+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 1.071 seconds
[2023-12-11T02:53:59.689+0000] {processor.py:154} INFO - Started process (PID=5868) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:53:59.691+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:53:59.692+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:53:59.692+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:54:00.422+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:54:00.723+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:54:00.722+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:54:00.760+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:54:00.760+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:54:00.790+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 1.105 seconds
[2023-12-11T02:54:30.890+0000] {processor.py:154} INFO - Started process (PID=5924) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:54:30.891+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:54:30.893+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:54:30.892+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:54:31.559+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:54:31.873+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:54:31.872+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:54:31.932+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:54:31.931+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:54:31.966+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 1.080 seconds
[2023-12-11T02:55:02.504+0000] {processor.py:154} INFO - Started process (PID=5989) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:55:02.506+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:55:02.508+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:55:02.507+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:55:03.143+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:55:03.427+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:55:03.426+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:55:03.465+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:55:03.465+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:55:03.497+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.997 seconds
[2023-12-11T02:55:34.234+0000] {processor.py:154} INFO - Started process (PID=6054) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:55:34.235+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:55:34.237+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:55:34.237+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:55:34.933+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:55:35.255+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:55:35.255+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:55:35.301+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:55:35.300+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:55:35.332+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 1.105 seconds
[2023-12-11T02:56:05.905+0000] {processor.py:154} INFO - Started process (PID=6111) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:56:05.906+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:56:05.908+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:56:05.908+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:56:06.737+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:56:07.072+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:56:07.071+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:56:07.106+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:56:07.106+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:56:07.144+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 1.243 seconds
[2023-12-11T02:56:37.961+0000] {processor.py:154} INFO - Started process (PID=6176) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:56:37.962+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:56:37.964+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:56:37.964+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:56:38.721+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:56:39.014+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:56:39.013+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:56:39.054+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:56:39.053+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:56:39.086+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 1.132 seconds
[2023-12-11T02:57:09.880+0000] {processor.py:154} INFO - Started process (PID=6232) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:57:09.882+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:57:09.883+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:57:09.883+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:57:10.515+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:57:10.776+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:57:10.775+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:57:10.809+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:57:10.808+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:57:10.843+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.968 seconds
[2023-12-11T02:57:41.181+0000] {processor.py:154} INFO - Started process (PID=6297) to work on /opt/airflow/dags/user_processing.py
[2023-12-11T02:57:41.183+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-12-11T02:57:41.184+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:57:41.184+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-12-11T02:57:41.858+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-12-11T02:57:42.172+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:57:42.171+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-12-11T02:57:42.212+0000] {logging_mixin.py:120} INFO - [2023-12-11T02:57:42.212+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-12-10T00:00:00+00:00, run_after=2023-12-11T00:00:00+00:00
[2023-12-11T02:57:42.243+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 1.067 seconds
